src/database/index.tsx

```diff
@@ -112,3 +117,52 @@ export const getSavedState = async (): Promise<EpidemicModelPersistent | null> =
     return null;
   }
 };
+
+const getBaselineScenario = async () => {
```

Comment:
I see this PR is marked as closing #114 but it looks like it only has support for a single scenario. Is that coming in a separate issue or did we move multiple scenarios out of this release?